### PR TITLE
Check for PSR2 & PSR-12 coding standards with PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by lucasmichot**
Check for PSR2 coding standards with PHP-CS-Fixer

- https://github.com/lucasmichot/podio-php/commit/2ad658e13e8ecab3b73627346494bcb31f452cb4:  Set php-cs-fixer to check for PSR2 rules.
- https://github.com/lucasmichot/podio-php/commit/7474619d4099198caa2628fc9062d6f625e2038e: Add a github action to check php-cs-fixer syntax.
- https://github.com/lucasmichot/podio-php/commit/e5a234307d2d0cc3819942fad05270951d243bfa: Fix all files

----

Also :
- https://github.com/podio-community/podio-php/pull/209/commits/b8fe4ace951950403886c79c188c1474c6ba323b : add PRR-12 ruleset
- https://github.com/podio-community/podio-php/pull/209/commits/fe9ae752bd35387eafa6d5595c37d1bd0b5c524f : fix all files

### CODE REVIEW

**Review by daniel-sc — APPROVED** [review]
Thanks for the contribution @lucasmichot! This is a good step forward. 
If you have time to even support PSR-12, I'd like to include that, but if you don't have time for that, it is fine for me as well!

File: .php-cs-fixer.dist.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+use PhpCsFixer\Config;
+use PhpCsFixer\Finder;
+
+$rules = [
+    '@PSR2' => true,
```
[the remark this answers is not in the document]
Comment (lucasmichot) [reply .php-cs-fixer.dist.php:7]: Sure 👍

**Comment by lucasmichot** [comment]
> Thanks for the contribution @lucasmichot! This is a good step forward.
> If you have time to even support PSR-12, I'd like to include that, but if you don't have time for that, it is fine for me as well!

Hey @daniel-sc , I have added an extra PSR-12 ruleset check. Feel free to review this PR again

**Review by daniel-sc — APPROVED** [review]
Perfect - thanks for the ultra fast updat!

**Review by rafaelmb — CHANGES_REQUESTED** [review]
Do we mention PSR-2 and 12 on our contribution guide? I think we should start in this PR.

Besides that, thank you lucas for the contribution!

**Comment by daniel-sc** [comment]
> Do we mention PSR-2 and 12 on our contribution guide? I think we should start in this PR.

@rafaelmb as we require "All build checks are passing", this is basically given? Of course, one could go for a more explicit description - but this could as well be confusing/demotivating for people not familiar with PSR-2/12?

**Comment by rafaelmb** [comment]
This is better

**Review by rafaelmb — APPROVED** [review]

**Review by rafaelmb — APPROVED** [review]
LGTM

**Pull request merged**